libsolidity/ast/AST.h

```diff
@@ -683,13 +683,13 @@ class UsingForDirective: public ASTNode
 	TypeName const* typeName() const { return m_typeName.get(); }
 
 	/// @returns a list of functions or the single library.
-	std::vector<ASTPointer<IdentifierPath>> const& functionsOrLibrary() const { return m_functions; }
+	std::vector<ASTPointer<IdentifierPath>> const& functionsOrLibrary() const { return m_functionsOrLibrary; }
 	bool usesBraces() const { return m_usesBraces; }
 	bool global() const { return m_global; }
 
 private:
 	/// Either the single library or a list of functions.
-	std::vector<ASTPointer<IdentifierPath>> m_functions;
+	std::vector<ASTPointer<IdentifierPath>> m_functionsOrLibrary;
```

Comment:
This can be a library rather than a list of functions and it's very easy to forget about that seeing it used. These two cases usually have to be handled differently.